### DISCUSSION TITLE
[FAQ entry] - difference to UB?

### DISCUSSION THREAD

**Discussion by rubyFeedback** [Spellhold-Studios/Almaterias-Restoration-Project#5] [Suggestions]
> In recent iterations it evolved into the younger, brasher sidekick of Unfinished Business.

Hi there,

Could the main README answer differences towards UB? In particular if we don't have
to install UB if we want to install ARP or vice versa, or whether ARP is an add-on (and
UB has to be installed first; or not). I think this may help people who want to install
ARP and check it out.

**Comment by Frenzgyn** [comment]
Hi,
Almateria's Restoration Project is not "redundant" with UB, it's an "unfinished business" mod that address things not addressed in UB. 

From the author's themselves, it's created with UB in mind, so it is better to install it AFTER UB, but it doesn't need or is needed by UB to work, so one could also install it WITHOUT UB.
 
The only thing not reccomended is to install it BEFORE UB, but honestly I did and I can't spot any strange issues, but better saffe than sorry. 

You're right that this kind of information could be added to the readme.
